be/src/exec/pipeline/set/union_const_source_operator.cpp

```diff
@@ -36,8 +38,9 @@ StatusOr<ChunkPtr> UnionConstSourceOperator::pull_chunk(starrocks::RuntimeState*
             // Each const_expr_list is projected to ONE dest row.
             DCHECK_EQ(_const_expr_lists[_next_processed_row_index + row_i].size(), columns_count);
 
+            ChunkPtr dummyChunk = ChunkHelper::createDummyChunk();
             ASSIGN_OR_RETURN(ColumnPtr src_column,
-                             _const_expr_lists[_next_processed_row_index + row_i][col_i]->evaluate(nullptr));
+                             _const_expr_lists[_next_processed_row_index + row_i][col_i]->evaluate(dummyChunk.get()));
```

Comment:
it seems that a empty_chunk is just OK;
```
ChunkPtr empty_chunk = std::make_shared<Chunk>();
```

Reply:
StatusOr<ColumnPtr> ExprContext::evaluate(Expr* e, Chunk* chunk, uint8_t* filter) {
    DCHECK(_prepared);
    DCHECK(_opened);
    DCHECK(!_closed);
#ifndef NDEBUG
    if (chunk != nullptr) {
        chunk->check_or_die();
        CHECK(!chunk->is_empty());
    }
    There is a check, do you have any context why?

Reply:
> it seems that a empty_chunk is just OK;
> 
> ```
> ChunkPtr empty_chunk = std::make_shared<Chunk>();
> ```

StatusOr ExprContext::evaluate(Expr* e, Chunk* chunk, uint8_t* filter) {
DCHECK(_prepared);
DCHECK(_opened);
DCHECK(!_closed);
#ifndef NDEBUG
if (chunk != nullptr) {
chunk->check_or_die();
CHECK(!chunk->is_empty());
}
There is a check, do you have any context why?